SensusService/Probes/PollingProbe.cs

```diff
@@ -208,7 +237,33 @@ protected override void InternalStart()
 
                     }, GetType().FullName + " Poll", TimeSpan.FromMinutes(_pollingTimeoutMinutes), userNotificationMessage);
 
+#if __IOS__
+                if (_significantChangePoll)
+                {
+                    _locationManager.RequestAlwaysAuthorization();
+                    _locationManager.DistanceFilter = 5.0;
+                    _locationManager.PausesLocationUpdatesAutomatically = true;
+                    _locationManager.AllowsBackgroundLocationUpdates = true;
+
+                    if (CLLocationManager.LocationServicesEnabled)
+                    {
+                        _locationManager.StartMonitoringSignificantLocationChanges();
+                    }
+                    else
+                    {
+                        SensusServiceHelper.Get().Logger.Log("Location services not enabled; please enable in Settings.", LoggingLevel.Normal, GetType());
+                    }
+
+                    _locationManager.LocationsUpdated += (o, e) => SensusServiceHelper.Get().ScheduleOneTimeCallback(callback, 0);
```

Comment:
If we're failing to get significant-change polls when the app is inactive, this might be why. `ScheduleOneTimeCallback` eventually starts a new invoke on the main thread (see `iOSSensusServiceHelper`), which will hang until the app returns to foreground. Create a new issue to resolve this, since the fix lies in `iOSSensusServiceHelper`.